app/components/FieldString/FieldString.css

```diff
@@ -44,3 +14,27 @@
 .list-link:hover {
   border-bottom-color: var(--edit-color-underline-hover);
 }
+
+.with-ellipsis {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+.free-input-wrapper {
+  display: flex;
+  margin: -10px 0 0 -10px;
+}
+
+.free-input-wrapper > * {
+  margin: 10px 0 0 10px;
+}
+
+.free-input-wrapper input {
+  flex: 1;
+}
+
+#open-icon {
```

Comment:
Did you mean to leave this ID selector?

Reply:
I didn't get to this yet, I'll do minor cleanups in the end (or in this case, when I switch them to the icon components)